cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -266,4 +266,9 @@
     </description>
   </property>
 
+  <property>
+    <name>metadata.storage.implementation</name>
```

Comment:
why is this change needed ?

Reply:
Oh that was for running elastic search for testing on my computer. I can get rid of that for the next commit.